### PR TITLE
Fix bug with AboutActivity crashing

### PR DESCRIPTION
**Pull request by HenriqueRocha**
I've noticed that in the latest develop branch the AboutActivity was crashing. After some investigation it was because two of the fragments were using one of the "gdl" layouts that have been removed.

I've added an Espresso test to verify that the About screen can actually be started (which failed of course) and fixed the problem until I got green. I've also done some code cleanup, reformatting and minor refactoring.

### CODE REVIEW

**Comment by tasomaniac** [comment]
Should we add monkey after every build on Travis? 
I did it in our company on Jenkins and it is really helpful.

File: app/src/main/java/org/gdg/frisbee/android/fragment/ChangelogFragment.java [hunk 1]
```diff
@@ -24,50 +24,36 @@
...
     @Override
     public void onViewCreated(View view, Bundle savedInstanceState) {
-        super.onViewCreated(view, savedInstanceState);    //To change body of overridden methods use File | Settings | File Templates.
+        super.onViewCreated(view, savedInstanceState);
 
-        InputStream is = null;
         try {
-            is = getResources().getAssets().open("changelog.html");
+            InputStream is = getResources().getAssets().open("changelog.html");
             mChangelog.setText(Html.fromHtml(Utils.inputStreamToString(is)));
```
Comment (tasomaniac) [inline app/src/main/java/org/gdg/frisbee/android/fragment/ChangelogFragment.java]: Is `is.close()` necessary?

**Comment by tasomaniac** [comment]
It looks really good. Really good refactoring. 
I am also trying to integrate checkstyle in my local branch. I also did lots of these :)

How did the PR where you removed GDL layout build? It should've been broken I think.

**Comment by HenriqueRocha** [comment]
@tasomaniac I didn't break because I still had two files missing in other folders with the same name. I need to make sure that I track all the usages next time. As a "self-punishment" I added an Espresso test to make sure that I don't break the About again. :D

**Comment by tasomaniac** [comment]
I see :)

File: app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java [hunk 2]
```diff
@@ -27,20 +27,16 @@
 import com.viewpagerindicator.TitlePageIndicator;
 
 import org.gdg.frisbee.android.R;
-import org.gdg.frisbee.android.fragment.*;
+import org.gdg.frisbee.android.fragment.AboutFragment;
+import org.gdg.frisbee.android.fragment.ChangelogFragment;
+import org.gdg.frisbee.android.fragment.ContributorsFragment;
+import org.gdg.frisbee.android.fragment.ExtLibrariesFragment;
+import org.gdg.frisbee.android.fragment.GetInvolvedFragment;
+import org.gdg.frisbee.android.fragment.TranslatorsFragment;
```
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java]: :+1:

File: app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java [hunk 3]
```diff
@@ -61,22 +55,20 @@ public void onCreate(Bundle savedInstanceState) {
...
 
-        mViewPagerAdapter = new AboutPagerAdapter(this, getSupportFragmentManager());
-        mViewPager.setAdapter(mViewPagerAdapter);
+        mViewPager.setAdapter(new AboutPagerAdapter(this, getSupportFragmentManager()));
         mIndicator.setViewPager(mViewPager);
     }
 
     protected String getTrackedViewName() {
-        return "About/"+getResources().getStringArray(R.array.about_tabs)[getCurrentPage()];
+        return "About/" + getResources().getStringArray(R.array.about_tabs)[getCurrentPage()];
```
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java]: Wow, that is some dirty code.

File: app/src/main/java/org/gdg/frisbee/android/fragment/ChangelogFragment.java (hunk 1 above)
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/fragment/ChangelogFragment.java]: Should be in a finally clause.

File: app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java (hunk 3 above)
Comment (HenriqueRocha) [inline app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java]: True, but I didn't want to change that as part of this PR. The change is only due to code reformatting. I will eventually refactor that code also. :)

Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java]: Yeah, it wasn't meant against you. I was just "admiring" the codebase.

Comment (mauimauer) [inline app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java]: *cough* *cough*

**Comment by HenriqueRocha** [comment]
@stefanhoth This should be ready to merge once the build is finished.
I even added a commit to close the stream that you mentioned.
After this one is merged I'll rebase the ViewPagerIndicator branch.

**Comment by HenriqueRocha** [comment]
Done. Should build successfully. :+1:

File: app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java (hunk 3 above)
Comment (stefanhoth) [inline app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java]: We live, we code, we learn. Even with this public code you got into Google. :tongue: jk, you did a fine job with this app but now it's time to complain about the style of each line. That's how it goes. :smirk:

**Comment by stefanhoth** [comment]
:evergreen_tree: so let's :package: :arrow_right:

**Pull request merged**